pkgs/by-name/co/codipack/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  cmake,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "codipack";
+  version = "2.3.1";
+
+  src = fetchFromGitHub {
+    owner = "SciCompKL";
+    repo = "CoDiPack";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-ZD9P4yWcF9zGeTyw6ENAcGoPyc8QcBdNZNnqRV4MH8s=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  meta = with lib; {
```

Comment:
remove with lib; as it increases eval times

Reply:
Done.